### PR TITLE
Conversion to Swift 4.2

### PR DESCRIPTION
**Pull request by andre991**
Conversion to swift 4.2

Issue #42

### CODE REVIEW

**Comment by andre991** [comment]
@ink-spot @SirArkimedes @0ber

**Comment by itruf** [comment]
+1

This fix is required to build for Swift 4.2

**Comment by drewpitchford** [comment]
Can we get this merged? I love the pod...but need the update.

**Comment by paskalem** [comment]
I don’t know if I can be of any help, but I was thinking that maybe It wasn’t a coincidence if I got this email , 
Sincerely yours

**Comment by rajeshbeats** [comment]
This is indeed update..

**Comment by paskalem** [comment]
🤞

File: UPCarouselFlowLayoutDemo.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -402,11 +419,13 @@
 			buildSettings = {
 				ALWAYS_EMBED_SWIFT_STANDARD_LIBRARIES = YES;
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
+				DEVELOPMENT_TEAM = X4EYDGL7NV;
```
Comment (SirArkimedes) [inline UPCarouselFlowLayoutDemo.xcodeproj/project.pbxproj]: It looks like there previously wasn't a development team set, should this be here?

File: UPCarouselFlowLayout/UPCarouselFlowLayout.swift [hunk 2]
```diff
@@ -60,7 +59,7 @@ open class UPCarouselFlowLayout: UICollectionViewFlowLayout {
         
         let yInset = (collectionSize.height - self.itemSize.height) / 2
         let xInset = (collectionSize.width - self.itemSize.width) / 2
-        self.sectionInset = UIEdgeInsetsMake(yInset, xInset, yInset, xInset)
+        self.sectionInset = UIEdgeInsets.init(top: yInset, left: xInset, bottom: yInset, right: xInset)
```
Comment (SirArkimedes) [inline UPCarouselFlowLayout/UPCarouselFlowLayout.swift]: I think the `.init` part shouldn't be here, as it looks like the isn't used in other places.

**Comment by drewpitchford** [comment]
If everyone agrees with the changes made, can we merge this? I think it is good to go.

**Comment by paskalem** [comment]
Okay let’s try it

**Comment by ink-spot** [comment]
Sorry guys I've been very busy lately and I just got a chance to look at that.
Thanks a lot for your contribution, I'll merge it right away!

**Pull request merged**